File: app/assets/javascripts/models/player.js
var app = app || {};

app.Player = Backbone.Model.extend({
  urlRoot: '/players',
  defaults: {
    team_id: null,
    first_name: '',
    last_name: '',
    age: 0,
    nationality: '',
    attack_rating: 0,
    defence_rating: 0,
    value: 0,
    position: '',
    goals: 0,
    assists: 0,
    yellow_cards: 0,
    red_cards: 0,
    mom_count: 0,
    injured: false
  }
});

File: app/models/player.rb
# == Schema Information
#
# Table name: players
#
#  id             :integer          not null, primary key
#  team_id        :integer
#  first_name     :string
#  last_name      :string
#  age            :integer
#  nationality    :string
#  attack_rating  :integer
#  defence_rating :integer
#  value          :integer
#  position       :string
#  goals          :integer          default(0)
#  assists        :integer          default(0)
#  yellow_cards   :integer          default(0)
#  red_cards      :integer          default(0)
#  mom_count      :integer          default(0)
#  injured        :boolean          default(FALSE)
#  created_at     :datetime         not null
#  updated_at     :datetime         not null
#

class Player < ActiveRecord::Base
  belongs_to :team
end
